calcom/cal.com
# fix: Managed Event Type Webhooks

## What does this PR do?

This PR fixes the remaining webhook events not being triggered for managed team events, namely:  Booking canceled, Booking rejected, Booking confirmed, and Recording ready

Fixes #10259

Have used a similar approach to my previous pull request #10025 

## Type of change


## How should this be tested?



- Set Up a managed event type
- Subscribe to all webhook events.
- Perform booking creation, cancelation, rescheduling, confirmation, and rejection events
- Check that webhook is being triggered for all of them.

## Mandatory Tasks
